__tests__/users.ts

```diff
@@ -2047,6 +2143,280 @@ describe('mutation updateUserProfile', () => {
       'GRAPHQL_VALIDATION_FAILED',
     );
   });
+
+  it('should validate github handle', () => {
+    const valid = [
+      'lee',
+      'github.com/lee',
+      'https://github.com/lee',
+      'https://github.com/lee/',
+    ];
+    const invalid = [
+      'lee#',
+      'http://github.com/lee',
```

Comment:
Considering this one or the one with queryparams maybe lets log some fails if need be we can just abstract these ourselves (but doubt they happen much)

Reply:
We should see increase in validation errors, I added `warn` log.